redwood-treehouse-host/src/commonMain/kotlin/app/cash/redwood/treehouse/TreehouseApp.kt

```diff
@@ -125,10 +124,14 @@ public class TreehouseApp<A : AppService> private constructor(
    * Continuously polls for updated code, and emits a new [LoadResult] instance when new code is
    * found.
    */
+  @OptIn(ExperimentalStdlibApi::class)
```

Comment:
What is this for? Whatever it is, it needs added to the README's list of unstable API usage.

Reply:
It’s for this:

```
    val dispatcher = dispatchers.zipline[CoroutineDispatcher.Key]
      ?: error("expected TreehouseDispatchers.zipline to include a CoroutineDispatcher")
```

But I get rid of this in a follow-up on the stack because I found a better way!

Reply:
I also forgot what repo I'm in. Zipline has the readme table. We can tolerate instability for now anyway.